conf/zapi/cdot/9.8.0/volume.yaml

```diff
@@ -57,6 +57,8 @@ counters:
 plugins:
   LabelAgent:
     value_mapping: status state online `1`
+    # metric label zapi_value rest_value `default_value`
+    value_to_num: new_status state online online `0`
```

Comment:
does dashboard also needs to change or also include new_status value?

Reply:
Yes, dashboard also need to be changed to consume new_status metric, it will be in next PR.